src/util/StringUtils.cpp

```diff
@@ -5,6 +5,9 @@
 
 #include "util/StringUtils.h"
 
+#include <unicode/bytestream.h>
+#include <unicode/casemap.h>
```

Comment:
I think you don't need them anymore with your CTRE implementation.

Reply:
I moved them for `StringUtils.h` to `StringUtils.cpp`, to remove uneccessary imports from the .h file. They are required by some other functions.